### PR TITLE
Minor website enhancements

### LINKED ISSUE #81: Minor website enhancements
- Add PHUSE EU to "Upcoming" section
- Update flip cards with names
- Update FAQ placeholder
- Fix screenshot file paths on Getting Started page
- Check spelling/grammar in all files

### PR DESCRIPTION
**Pull request by edelarua**
Closes #81

### CODE REVIEW

File: index.qmd [hunk 1]
```diff
@@ -48,7 +47,8 @@ The falcon initiative is an industry collaborative effort under pharmaverse that
 :::
 
 ::: flip-card-back
-Roche
```
Comment (mattkorb) [inline index.qmd:51]: Do we also want to list the POs or only the developer?

**Review by mattkorb — APPROVED** [review]
Hi @edelarua 
only one small question, besides that all looks good to me :)

File: index.qmd (hunk 1 above)
Comment (edelarua) [reply index.qmd:51]: Hi @mattkorb, for now I think just developer names and if Vincent wants to add PO names as well he will probably add them within his branch.  Thanks! :)

Comment (mattkorb) [reply index.qmd:51]: Sounds good 👍

**Pull request merged**